MIGRATING_to_v4.md

```diff
@@ -0,0 +1,14 @@
+# Migrate to Design System version 4
```

Comment:
Should we start moving the migration guides and similar to folders now since we are getting quite a few of them?

Reply:
Probably wouldn't hurt